packages/firebase_vertexai/firebase_vertexai/lib/src/vertex_schema.dart

```diff
@@ -0,0 +1,207 @@
+// Copyright 2024 Google LLC
```

Comment:
[nit] I don't think the `vertex_` prefix in the file name adds value. Consider renaming this as `lib/src/schema.dart`, or `lib/src/json_schmea.dar`.